### PR TITLE
i18n(zh-cn): Update ghost.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update ghost.mdx

#### Related issues & labels (optional)

- #6436

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/ghost.mdx [hunk 1]
```diff
@@ -282,8 +282,9 @@ const { post } = Astro.props;
 要部署你的网站，请访问我们的[部署指南](/zh-cn/guides/deploy/)，并按照你更喜欢的托管服务提供商的部署说明进行操作。
 ## 社区资源 
 
-- 你也可以从这个 [Astro Ghost 启动器](https://github.com/PhilDL/astro-starter-ghost) 开始集成你的 Ghost 与 Astro。
+- GitHub 上 [`astro-starter-ghost`](https://github.com/PhilDL/astro-starter-ghost)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/ghost.mdx:285]:
```suggestion
- GitHub 上的 [`astro-starter-ghost`](https://github.com/PhilDL/astro-starter-ghost)
```

File: src/content/docs/zh-cn/guides/cms/ghost.mdx [hunk 2]
```diff
@@ -282,8 +282,9 @@ const { post } = Astro.props;
...
+- GitHub 上 [`astro-ghostcms`](https://github.com/MatthiesenXYZ/astro-ghostcms)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/ghost.mdx:286]:
```suggestion
- GitHub 上的 [`astro-ghostcms`](https://github.com/MatthiesenXYZ/astro-ghostcms)
```

File: src/content/docs/zh-cn/guides/cms/ghost.mdx [hunk 3]
```diff
@@ -282,8 +282,9 @@ const { post } = Astro.props;
...
 - [Astro + Ghost + Tailwind CSS](https://andr.ec/posts/astro-ghost-blog/) 由 Andre Carrera 提供
-- [Ghost CMS & Astro](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) 由 Adam Matthiesen 提供
+- [Ghost CMS & Astro Tutorial](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) 由 Adam Matthiesen 提供
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/ghost.mdx:288]:
```suggestion
- [Ghost CMS & Astro 教程](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) 由 Adam Matthiesen 提供
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by dreyfus92 — APPROVED** [review]
Thanks peeps, LGTM ✅

**Pull request merged**